dotenv/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "dotenv"
```

Comment:
My main idea is to divide the project into several internal crates (without publishing them to crates.io) and improve the code quality.

> Looks like this crate is intended to contain logic around dotenv files.
I think we need to choose more appropriate name for this one. Moreover this name is already exists on crates.io ([link](https://crates.io/crates/dotenv)) so it may be misleading.

I don't have plans to publish this crate to crates.io, but maybe you're right. Do you have any ideas how to name this internal crate?

> Use cargo workaspaces to be able to easily extend number of crates without extra needs to jump to dir to use cargo commands (like test/clippy/fmt). It also useful for CI.

Yes. It'll be the next step.

Reply:
> I don't have plans to publish this crate to crates.io, but maybe you're right.

In this case, it shouldn't work, because all dependencies should be accessible during publishing to `crates.io` (we publish `dotenv-linter` itself)

Reply:
> Do you have any ideas how to name this internal crate?

Let's firstly discuss what do we want to keep there? For example, `LineEntry` has some fields related to specific checkers/fixers. It's not a problem, but we need to make it more clear & independent. 

In general, it looks like `dotenv-source` or `dotenv-lookup` could be an appropriate name. Because currently it works only with files, but probably we might extend it to work with pure-strings or streams.

Reply:
> In this case, it shouldn't work AFAIK, because all dependencies should be accessible during publishing to `crates.io` (we publish `dotenv-linter` itself)

Yes, you're right. I forgot about it 😅

Reply:
> Let's firstly discuss what do we want to keep there? For example, `LineEntry` has some fields related to specific checkers/fixers. It's not a problem, but we need to make it more clear & independent. So I guess the main sense here is to extract line-entries from some source?
>
> In general, it looks like `dotenv-source` or `dotenv-lookup` could be an appropriate name. Because currently it works only with files, but probably we might extend it to work with pure-strings or streams.

I like `dotenv-lookup` 👍